man/systemd-vmspawn.xml

```diff
@@ -507,6 +507,20 @@ $ mkosi -d arch -p systemd -p linux --autologin -o image.raw -f build
 $ systemd-vmspawn --image=image.raw
       </programlisting>
     </example>
+
+    <example>
+      <title>Import and run a Fedora 39 Cloud image using machinectl</title>
+
+      <programlisting>
+$ wget https://download.fedoraproject.org/pub/fedora/linux/releases/39/Cloud/x86_64/images/Fedora-Cloud-Base-39-1.5.x86_64.raw.xz \
+       https://download.fedoraproject.org/pub/fedora/linux/releases/39/Cloud/x86_64/images/Fedora-Cloud-39-1.5-x86_64-CHECKSUM
```

Comment:
Please use `&fedora_cloud_release;` here. I.e. partially revert 19e980eb92d923a25f0cbc38b945c282bc5f5b1b and see what was done before fd571c9df0ba86636401582c4ea0a8cb608e1216.

Hmm, using both `curl` and `wget` seems a bit iffy. I think we prefer `curl`, because it's more modern and better supported.

EDIT: Wow, curl does not allow two files to be downloaded, and if you don't specify `-L`, it just silently writes an empty files? Sweet Jesus. So maybe use `wget` everywhere.

Reply:
so curl actually does support downloading two files, I actually got it to work but for some reason it didn't work for these two in particular???
``curl -O https://mirrors.edge.kernel.org/archlinux/iso/latest/sha256sums.txt -O https://mirrors.edge.kernel.org/archlinux/iso/latest/archlinux-bootstrap-x86_64.tar.gz.sig``
This curl command for me downloads both files, but doing it for the fedora ones just didn't work??

Reply:
So I got a curl command to work and wget started being funny when it tried to download all three files with it so I've just gone with a single curl command.